src/home.js

```diff
@@ -0,0 +1,36 @@
+const mainDiv = document.getElementById('content');
+const homeSlate = document.createElement('div');
+homeSlate.classList = 'homeSlate blank';
+
+export function navigation() {
+	const navList = document.createElement('ul');
+	navList.innerHTML = `
+        <li><a id="home" href="#home">Home</a></li>
+        <li><a id="menu" href="#menu">Menu</a></li>
+        <li><a id="about" href="#about">About</a></li>
+        <li><a id="contact" href="#contact">Contact</a></li>
+    `;
+	mainDiv.appendChild(navList);
+}
+
+export function homePage() {
+	const header = document.createElement('h1');
+	header.innerHTML = '<small>welcome</small> To Our Restaurant!';
+	homeSlate.appendChild(header);
+	mainDiv.appendChild(homeSlate);
+}
```

Comment:
### Fix the bearing bug
- [ ] We have a bug in our code. Whenever a user clicks on the links provided for in the navigation bar, the contents of the other modules get piled up to the top of one another in the final result. The user will be forced to refresh the page before clicking on the page again to see its contents only. This is a flow and goes against the original objective of the project - to have a modularised codebase with independent units. Kindly fix this. This is what we currently have 👇

![bug-restaurant](https://user-images.githubusercontent.com/22345961/101300936-4ddc0e00-3848-11eb-94f2-38cf3bdd987b.png)